### ISSUE TITLE
General Error Issue 

### ISSUE THREAD

**Issue by IliaMaz** [fruitcake/laravel-cors#448]
Problem: So as a pretty new developer, it was hard for me to understand the error message being thrown when "my/your" backend code is incorrect. This had caused me to waste days and the same could probably be said for other countless other devs. Errors in backend code will result in either: No Access Origin Control OR when you add a wildcard origin header to app.php it throws multiple of the same cannot be present.
Solution:
Do your error messages correctly if possible.
|
Make a notice on the front page readme about this issue.
I've not seen this problem mentioned in any of the newer issues, which is why I find it necessary to point and say that this is inexcusable

**Comment by barryvdh** [comment]
Can you provide an example what was wrong? Problem is that the response is swallowed by the browser when the headers missing.

**Comment by IliaMaz** [comment]
This is the example: https://github.com/fruitcake/laravel-cors/issues/447
The error was in my controller in laravel, it contained an error. But, as you can see on the photos there, the errors being thrown are not correct. 

tl;dr: backend code is wrong, console throws errors about cors, instead of throwing errors about my code. 

I don't know whether you could fix this error message. But it would be nice to know about this problem in the readme so that devs don't have to waste time trying to fix an error which is nonexistent.

**Comment by joke2k** [comment]
Hi @barryvdh @IliaMaz I debug this issue https://github.com/fruitcake/laravel-cors/issues/433#issuecomment-620902302 
I have not found any solution yet but I will.

**Comment by barryvdh** [comment]
I'm closing this due to inactivity. If you still encounter this issue on the latest version (2.0.1 at the moment), please open a new issue with detailed information.
